kitsune/users/migrations/0033_batch_delete_non_migrated_users.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations, transaction
+from django.db.models import Q
+
+
+def delete_non_migrated_users(apps, schema_editor):
+    """
+    Delete users where is_fxa_migrated is False and who aren't creators of 
+    any revision, question, or answer, and who don't have any votes or messages
+    """
+    User = apps.get_model('auth', 'User')
+    
+    with transaction.atomic():
```

Comment:
This is not needed. Django migrations are performed atomically by default.

Reply:
Thanks!